### ISSUE TITLE
Poor documentation

### ISSUE THREAD

**Issue by branc116** [LisacPisac/SIS-CORS-Projekt#1]
There isn't any documentation in this project.
I think you should add some docs to readme file.

**Comment by LisacPisac** [comment]
You are absolutely correct. Thank you for pointing this out. I shall do my best to add some quality documentation to this repository.
